src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/AlignmentDuplicateBQSRProgramGroup.java

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+
+public class AlignmentDuplicateBQSRProgramGroup  implements CommandLineProgramGroup {
```

Comment:
This is a confusing group.  Maybe "BamPreprocessing" would be less specific but more clear?

Reply:
Sure. Geraldine mentioned the Picard folks may not like the current category name so BAM Preprocessing will likely cause fewer protests.